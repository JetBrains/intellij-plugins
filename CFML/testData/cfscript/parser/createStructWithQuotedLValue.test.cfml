<cfscript>
  function foo(){
    var foo = { "bar" = "foo" };
  }
</cfscript>
