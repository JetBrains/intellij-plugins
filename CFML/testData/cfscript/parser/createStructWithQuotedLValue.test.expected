CfmlFile:createStructWithQuotedLValue.cfml
  Tag
    PsiElement(OPENER)('<')
    PsiElement(CF_TAG_NAME)('cfscript')
    PsiElement(R_ANGLEBRACKET)('>')
    FunctionDefinition
      PsiElement(FUNCTION_KEYWORD)('function')
      PsiElement(IDENTIFIER)('foo')
      ParametersList
        PsiElement(L_BRACKET)('(')
        PsiElement(R_BRACKET)(')')
      FUNCTIONBODY
        BLOCK_OF_STATEMENTS
          PsiElement(L_CURLYBRACKET)('{')
          Assignment
            PsiElement(VAR_KEYWORD)('var')
            ReferenceExpression
              PsiElement(IDENTIFIER)('foo')
            PsiElement(ASSIGN)('=')
            PsiElement(L_CURLYBRACKET)('{')
            Assignment
              PsiElement(DOUBLE_QUOTE)('"')
              PsiElement(STRING_TEXT)('bar')
              PsiElement(DOUBLE_QUOTE_CLOSER)('"')
              PsiElement(ASSIGN)('=')
              StringLiteral
                PsiElement(DOUBLE_QUOTE)('"')
                PsiElement(STRING_TEXT)('foo')
                PsiElement(DOUBLE_QUOTE_CLOSER)('"')
            PsiElement(R_CURLYBRACKET)('}')
          PsiElement(SEMICOLON)(';')
          PsiElement(R_CURLYBRACKET)('}')
    PsiElement(LSLASH_ANGLEBRACKET)('</')
    PsiElement(CF_TAG_NAME)('cfscript')
    PsiElement(CLOSER)('>')
